core/eolearn/core/eodata.py

```diff
@@ -689,14 +688,10 @@ def consolidate_timestamps(self, timestamps: List[dt.datetime]) -> Set[dt.dateti
         good_timestamp_idxs = [idx for idx, _ in enumerate(self.timestamps) if idx not in remove_from_patch_idxs]
         good_timestamps = [date for idx, date in enumerate(self.timestamps) if idx not in remove_from_patch_idxs]
 
-        for feature_type in [
-            feature_type for feature_type in FeatureType if (feature_type.is_temporal() and feature_type.has_dict())
-        ]:
+        relevant_features = set(FeatureTypeSet.TEMPORAL_TYPES) - {FeatureType.TIMESTAMPS}
+        for feature_type in relevant_features:
             for feature_name, value in self[feature_type].items():
-                if isinstance(value, np.ndarray):
-                    self[feature_type][feature_name] = value[good_timestamp_idxs, ...]
-                if isinstance(value, list):
-                    self[feature_type][feature_name] = [value[idx] for idx in good_timestamp_idxs]
```

Comment:
why was this even here? I guess it's just for timestamps, but redundant since they were set below anyway

Reply:
god knows :D